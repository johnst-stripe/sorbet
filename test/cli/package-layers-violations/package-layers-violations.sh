cd test/cli/package-layers-violations || exit 1

../../../main/sorbet --silence-dev-message --stripe-packages --stripe-packages-layers=lowest,middle,highest --max-threads=0 . 2>&1
