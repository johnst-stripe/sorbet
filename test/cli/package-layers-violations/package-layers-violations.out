layer-violating-pkg/__package.rb:7: Layering violation: package `LayerViolatingPackage` with layer `middle` imports package `HighestLayerPackage` which has layer `highest` https://srb.help/3718
     7 |  import HighestLayerPackage
                 ^^^^^^^^^^^^^^^^^^^
Errors: 1
